### PR TITLE
Add master/slave to the Conscious Language section

### PR DESCRIPTION
**Pull request by Amrita42**
This PR  adds the terms master/slave under the **Conscious Language** title and suggests alternatives as per the linked Conscious Language GitHub repo.

### CODE REVIEW

**Review by inoxx03 — DISMISSED** [review]

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (stoobie) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: Suggest changing "a primary" to "the primary".

Comment (stoobie) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: Also "a master copy" to "the master copy"

Comment (stoobie) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: I also suggest adding "main" as an example to replace the "master" branch in a repo.

**Review by stoobie — CHANGES_REQUESTED** [review]
@Amrita42 
I suggest moving line 21 above line 19, so that it applies to all terms, including the ones you are adding.
That, and my other comments are all I have to suggest.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: Added another example for **main**

Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc]: Done. added main example.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (stoobie) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:19]:
```suggestion
When possible, rewrite documentation to avoid these terms:

- blacklist
- whitelist
- master
- slave

```

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (stoobie) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:54]: Move this section to precede the examples. Right now, it scans that this section is part of the slave example.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (stoobie) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:23]: I suggest changing all terms such that they use italics to set off the term rather than single quotes.

**Review by stoobie — CHANGES_REQUESTED** [review]
A few more suggestions.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (stoobie) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:19]: This list doesn't make any sense without first listing the terms your suggesting alternatives *for*.

**Review by stoobie — CHANGES_REQUESTED** [review]
One comment.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc]: oops my bad . So similar to what alternatives has been suggested for blacklist/whitelist. I have added these alternatives under master/slave to keep t consistent. what do you think?

Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc]: @stoobie

**Comment by porueesq** [comment]
@redhat-documentation/ccs-style-council :  This is ready for more reviews. Can you please take a look?

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (ndeevy) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:50]: Great addition to the guide here. My only comment is around the word 'using', which can produce a double translation when it's used like this. We could fix it by switching the example around:

no:
Use the following command to copy the public key to the _slave_ node.

yes:
Use the following command to copy the public key to the _secondary_ node.

Or we could add 'by' before 'using' to fix it also.

**Comment by ndeevy** [comment]
Looks great @Amrita42  just one comment/suggestion :-]

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc]: @Amrita42 

The construction of the page still doesn't make sense to me when I scan it. And the suggested alternatives appear to be part of the second example in both headings. 

I again suggest:

When possible, rewrite documentation to avoid these terms:
- blacklist
- whitelist
- master
- slave
=== Blacklist and whitelist
<Suggest alternatives>
Example 1
Example 2
=== Master and slave
<Suggest alternatives>
Example 1
Example 2

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:62]: Sounds good :)

**Comment by Amrita42** [comment]
@stoobie  @ndeevy  @inoxx03  PTAL at the modified PR and if all looks good , approve.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (dfitzmau) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: This list seems restrictive. What about parent/child? Maybe the introductory statement to the list should mention that the following are examples of alternative terms. WDYT?

I think a note stating that one should consult with their project team to agree on a selection of alternative terms that would suit the upstream and downstream community of a certain product would not go amiss. I have had engineers and QE express concern that alternative terms were selected without their input.

Comment (ndeevy) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: The conscious language group have offered these options as ".. teams should follow the recommendations outlined by the Linux kernel. These recommendations have informed the terminology recommendations we provide below.

- primary / secondary
- source / replica
- initiator, requester / responder
- controller, host / device, worker, proxy
- director / performer
"
I agree about the consulting with engineering teams. In the previous entry about black/white list, I added this line "Use this combination unless your product area has another specific replacement that is agreed between engineering leadership and your documentation team." -so we could include something like that more explicitly, I guess at the top of this section wold make sense.

The conscious language group are clear that we need to consult with our product teams, so you could escalate the query up to product management, who might have already fixed on replacement terms but it wasn't communicated as far as docs yet.

This is a bit from the **Inclusive Language Project: FAQ** on the Source:
"To ensure consistency and success, it is imperative for product team stakeholders to align internally. For example, documentation teams should engage in discussions with their engineering leadership to reach an agreement on replacement terms. This will ensure that the product documentation will match the code.

If your project opts to use some other term or phrase (for example, Fedora chose ‘rawhide’ as their replacement for their master branch) please let us know that you’ve done so, and why, so that we can learn from your decisions."

Comment (dfitzmau) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: Thanks, @ndeevy . Finding information on the Source can be difficult when people need to get docs out. I think a note at the start of this section would help that explains that a writer should seek stakeholder agreement when selecting a suitable alternative term for a specific product. I cannot see the start of this section on the PR, so maybe such a note already exists before the description of each term to replace.

Comment (stoobie) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: @dfitzmau @ndeevy There's a link to the Github repo for the Conscious Language Program in the PR's first paragraph. Jump to there and the same info appears.

**Review by stoobie — APPROVED** [review]
@Amrita42 LGTM, just check that the output renders as expected.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (dfitzmau) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: Thanks, @stoobie . I expanded the PR and found that link. I still had to go digging to find a statement about seeking agreement from project stakeholders. I still think having a short note at the start of the section would help writers avoid getting egg on their face when stakeholders raise concerns about alternative term choice. Getting stakeholder buy-in before adding a term to a product's docs would really alleviate any concerns on a product team.

Comment (stoobie) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: @dfitzmau Understood. I just wanted everyone to be aware that in this case, we don't have to depend on the Source being the source of truth, that there's a public site with the info. I think saying something about stakeholder collaboaration/managment is fine.

**Comment by Amrita42** [comment]
@ndeevy Implemented suggestions , please approve if all looks good.

@inoxx03 PTAL and approve if all looks good.

**Comment by stoobie** [comment]
> @ndeevy Implemented suggestions , please approve if all looks good.
> 
> @inoxx03 PTAL and approve if all looks good.

@Amrita42 Did you check how it renders? I'm worried that the examples under each bullet are not properly indented.

**Comment by Amrita42** [comment]
> > @ndeevy Implemented suggestions , please approve if all looks good.
> > @inoxx03 PTAL and approve if all looks good.
> 
> @Amrita42 Did you check how it renders? I'm worried that the examples under each bullet are not properly indented.

@stoobie om/stoobie Yes to me It looks good in the file view.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: Need a space after the /

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/general-formatting.adoc:51]: I think that we also need to add the clause about or any other combination that is approved for your product. For example, on OpenShift, our replacement is control plane / worker (or compute). S

**Review by bergerhoffer — COMMENTED** [review]
@Amrita42 Added some feedback, but wasn't able to review the examples too well since they aren't rendering properly.

It's a good idea to do a local build of the SSG (not just viewing the file in GitHub) to verify that everything is rendering properly. You can do so by running `asciidoctor master.adoc` and viewing the generated master.html file.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (stoobie) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:28]: @Amrita42 @bergerhoffer I am fine with either way of presenting. I tend to agree that the headings were clearer. You can fix the indentation issues by just adding + after the bulleted item and any paragraphs that you want to be indented, like this:
```
* bulleted item
+
paragraph
+
paragraph
```

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (dfitzmau) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: Hi @Amrita42 . Might I ask the decision on my query?

The statement on the GitHub source is perfect, but it's buried and does not stand out among the README text. 

Can we add this statement (that follows) as an important notice to the CCS guide? I personally experienced resistance from engineering when I replaced an undesired legacy term with one of the alternatives from the list on this PR.

_To ensure consistency and success, it is imperative for product team stakeholders to align internally. For example, documentation teams should engage in discussions with their engineering leadership to reach an agreement on replacement terms. This will ensure that the product documentation will match the code._

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:22]: Sounds good

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: @dfitzmau  imo I  think we can add a statement at the start of the section. Will add that.

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:51]: +1

**Comment by joaedwar** [comment]
[CCS Style Council](https://github.com/orgs/redhat-documentation/teams/ccs-style-council) This PR needs a few more project member reviews for final approval. Please review if you have the time. Thank you

**Review by joaedwar — DISMISSED** [review]

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:44]: Modified the sentence to reflect present tense

**Review by bergerhoffer — COMMENTED** [review]
One issue and one suggestion!

File: supplementary_style_guide/style_guidelines/general-formatting.adoc
Comment (Amrita42) [reply supplementary_style_guide/style_guidelines/general-formatting.adoc:50]: since we have a similar statement for whitelist/blacklist, I have added a similar statement to master and slave to keep consistency.

**Review by dfitzmau — APPROVED** [review]
Hi Amrita! Nice job on the content. LGTM.

**Review by ndeevy — APPROVED** [review]

**Review by stoobie — APPROVED** [review]
LGTM

**Review by bergerhoffer — APPROVED** [review]
LGTM now too. Merging!

**Pull request merged**
